rockcraft.yaml

```diff
@@ -32,40 +32,22 @@ services:
 platforms:
   amd64:
 
+# Please refer to
+# https://discourse.ubuntu.com/t/unifying-user-identity-across-snaps-and-rocks/36469
+# for more information about shared user.
+run_user: _daemon_
```

Comment:
This might be a nice place to note that `584792` belongs to this user. 

(IMO) It would be better if instead of the ID value you could pass the user/group name below. Do you know if we can do that?

Reply:
Unfortunately no, you get the error `Bad rockcraft.yaml content: value is not a valid integer`. But I've added a note as to the mapping between id and user here.